src/pydot/dot_parser.py

```diff
@@ -84,7 +84,7 @@ class HTML(Token):
     """Parsing for HTML-like strings."""
 
     def __init__(self) -> None:
-        super().__init__()
+        super().__init__()  # type: ignore
```

Comment:
This is only necessary in Python 3.8, and in later versions `mypy --strict` will actually complain about it. That's why I removed it in #464, and changed the CI definition to do MyPy checks in Python 3.9 and 3.13, instead of 3.8 and 3.13. Writing our type annotations for an unsupported Python release, at the expense of current releases, feels like the wrong way to break on that one.